src/App.vue

```diff
@@ -6,5 +6,7 @@
 import { useAppStore } from './store/app'
 
 const store = useAppStore()
-store.initLogin()
+
+const logToken = process.env.NODE_ENV === 'development'
```

Comment:
I'm not sure if we are there yet.
Because the docker deployment is in production
And then I have no way to get a token currently, which would make testing very very difficult.
Overall good idea, but let's maybe wait until we have a gateway with token in the frontend

Reply:
Alright, I will change that!